lib/bugcrowd_templates.rb

```diff
@@ -40,15 +40,52 @@ def get(
       file_name: file_name
     ).template_file(type)
 
-    template_data(template_path)
+    template_data(file_path: template_path, category: category, subcategory: subcategory, item: item)
```

Comment:
Yes `template_data` will call recursively. based on file_path it will check the file is exists or not, other wise it will check the parent folder to get the file. Can you please suggest me for  `structure it/naming things?`..

Reply:
I'm always nervous about recursive methods, it's really easy to fall into infinite calling. 
Maybe a loop instead? 
also the category/subcategory/item structure looks really tailored to the vrt. what happen with other templates if we have a different path structure? like methodologies notes ..

Reply:
updated the code based on category, subcategory & item